Meta/Lagom/Tools/CodeGenerators/LibWeb/WrapperGenerator/IDLGenerators.cpp

```diff
@@ -1345,6 +1351,18 @@ static void generate_wrap_statement(SourceGenerator& generator, String const& va
     } else if (type.name == "unsigned long") {
         scoped_generator.append(R"~~~(
     @result_expression@ JS::Value((u32)@value@);
+)~~~");
+    } else if (type.name == "long long") {
+        scoped_generator.append(R"~~~(
+    auto bigval = ::Crypto::SignedBigInteger::create_from((i64)@value@);
+    
+    @result_expression@ JS::Value(js_bigint(vm, bigval));
+)~~~");
+    } else if (type.name == "unsigned long long") {
+        scoped_generator.append(R"~~~(
+    auto bigval = ::Crypto::SignedBigInteger(::Crypto::UnsignedBigInteger::create_from((u64)@value@));
+
+    @result_expression@ JS::Value(js_bigint(vm, bigval));
```

Comment:
We're supposed to return the closest numeric double value for these instead of BigInts:
> The result of [converting](https://webidl.spec.whatwg.org/#dfn-convert-idl-to-ecmascript-value) an IDL [long long](https://webidl.spec.whatwg.org/#idl-long-long) value to an ECMAScript value is a Number value that represents the closest numeric value to the [long long](https://webidl.spec.whatwg.org/#idl-long-long), choosing the numeric value with an even significand if there are two [equally close values](https://tc39.es/ecma262/#sec-ecmascript-language-types-number-type). If the [long long](https://webidl.spec.whatwg.org/#idl-long-long) is in the range [−253 + 1, 253 − 1], then the Number will be able to represent exactly the same value as the [long long](https://webidl.spec.whatwg.org/#idl-long-long).

https://webidl.spec.whatwg.org/#es-long-long

> The result of [converting](https://webidl.spec.whatwg.org/#dfn-convert-idl-to-ecmascript-value) an IDL [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long) value to an ECMAScript value is a Number value that represents the closest numeric value to the [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long), choosing the numeric value with an even significand if there are two [equally close values](https://tc39.es/ecma262/#sec-ecmascript-language-types-number-type). If the [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long) is less than or equal to 253 − 1, then the Number will be able to represent exactly the same value as the [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long).

https://webidl.spec.whatwg.org/#es-unsigned-long-long